### DISCUSSION TITLE
RPG{LE} Content assist with /include

### DISCUSSION THREAD

**Discussion by ThomasBarberot** [codefori/vscode-ibmi#1072] [Q&A]
Hello.
I use a lot of members to be included in others, in particular to code prototypes, variables and constant values only once, and not in every member.
When I use an /include of a member  (ex. : MBRINC) in another (ex. : MBRPPL), in the member where the /include is set (MBRPPL), the content assist ([CTRL] + space) doesn"t seem to display "stuffs" (prototype or variable) from the included member (MBRINC).
Do I do something wrong or is it a limitation of Code for I ?
Thanks.

**Comment by DelYv** [comment]
I am facing a similar issue where I work with stream files, iASP, and grouped includes. To ensure my tooling is always available, I created a tool.rpgleinc file where I group my other includes such as number.rpgleinc and string.rpgleinc. RDI resolves the includes and proposes the procedures from the embedded includes, like the procedures from number.rpgleinc, but the code for I does not do it. Additionally, I have attached a screenshot for reference.

![javaw_23-08-30-09-17-36](https://github.com/halcyon-tech/vscode-ibmi/assets/114941503/cc435339-abd5-45b2-96c2-951c8dfd9b17)

 Thanks

**Reply by worksofliam** [reply]
Code for IBM i (well, vscode-rpgle specifically) does not resolve includes of includes currently.
